content/english/blog/2024-06-open-sauce.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "Pawprint Goes to OpenSauce (again!)"
+date: 2024-06-22
+author: rechner
+image : "images/blog/blog-post-2.jpg"
+bg_image: "images/blog/2024/paw-simon/banner.jpg"
+categories: ["Hackerspace News"]
+tags: ["Projects Showcase", "OpenSauce"]
+description: "A look at the projects and experiences of Maker Youtube's coolest place to show things off"
+draft: false
+type: "post"
+---
+
+June 15th marked Pawprint Prototyping's return to [OpenSauce](https://opensauce.live) - the whimsical convention
+for Science and Makers YouTubers, and brainchild of [William Osman](https://www.youtube.com/@williamosman).
```

Comment:
```suggestion
for Science and Maker YouTubers, and brainchild of [William Osman](https://www.youtube.com/@williamosman).
```

idk this might still not be right